modules/oadp-features-plugins-known-issues.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+// oadp-features-plugins-known-issues
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+:_content-type: CONCEPT
+[id="oadp-features-plugins-known-issues_{context}"]
+= OADP plugins known issues
+
+The following section describes known issues in {oadp-first} plugins:
+
+[id="velero-plugin-panic_{context}"]
+== Velero plugin panics on imagestream backup due to a missing secret
+
+The velero plugin panics on imagestream backup. A missing secret causes this panic error.
+
+When the backup and the Backup Storage Location (BSL) are managed outside the scope of the Data Protection Application (DPA), the OADP controller, meaning the DPA reconciliation does not create the relevant `oadp-<BSL Name>-<BSL Provider>-registry-secret`.
+
+When the backup is run, the openshift-velero-plugin panics on the imagestream backup, with a panic error:
+
+[source,terminal]
+----
+024-02-27T10:46:50.028951744Z time="2024-02-27T10:46:50Z" level=error msg="Error backing up item"
+backup=openshift-adp/<backup name> error="error executing custom action (groupResource=imagestreams.image.openshift.io,
+namespace=<BSL Name>, name=postgres): rpc error: code = Aborted desc = plugin panicked:
+runtime error: index out of range with length 1, stack trace: goroutine 94…
+----
+
+[id="velero-plugin-panic-workaround_{context}"]
+=== Workaround to avoid the panic error
+
+To avoid the velero plugin panic error, follow these steps:
+
+. Label the custom BSL with the relevant label
++
+[source,terminal]
+----
+$ oc label BackupStorageLocation <BSL Name> app.kubernetes.io/component=bsl
```

Comment:
Per the [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values), replaceable values take an underscore between words. Also, while not in the manual, we generally use lowercase in my experience. 

```suggestion
$ oc label BackupStorageLocation <bsl_name> app.kubernetes.io/component=bsl
```